src/spritekit.cs

```diff
@@ -1464,23 +1528,65 @@ interface SKUniform : NSCopying, NSCoding {
 		[Export ("initWithName:float:")]
 		IntPtr Constructor (string name, float /* float, not CGFloat */ value);
 
+		[Internal]
+		[Availability (Deprecated = Platform.iOS_10_0 | Platform.Mac_10_12)]
 		[Export ("initWithName:floatVector2:")]
-		IntPtr Constructor (string name, Vector2 value);
+		IntPtr InitWithNameFloatVector2 (string name, Vector2 value);
+
+		[Internal]
+		[iOS (10,0)][Mac (10,12)]
+		[Export ("initWithName:vectorFloat2:")]
+		IntPtr InitWithNameVectorFloat2 (string name, Vector2 value);
```

Comment:
Aaaaaaah indeed, I wasn't sure what that was for (it didn't come back to my mind when I saw these errors).

It looks like we specifically use that for `vector_float3`.